src/categories/categories.service.ts

```diff
@@ -23,4 +31,18 @@ export class CategoriesService {
   remove(id: number) {
     return `This action removes a #${id} category`;
   }
-}
+
+  async getAllApis(categoryId: string){
+    //check if category exists
```

Comment:
Nice one...

You can use a try-catch block here

Reply:
Thanks. Done